airbyte-integrations/connectors/source-camunda-history/source_camunda_history/spec.yaml

```diff
@@ -0,0 +1,40 @@
+connectionSpecification:
+  type: object
+  $schema: http://json-schema.org/draft-07/schema#
+  required:
+    - username
+    - start_date
+    - batchsize
+    - baseurl
+  properties:
+    baseurl:
+      type: string
+      order: 4
+      title: baseurl
+      description: http://localhost:8090
+    password:
+      type: string
+      order: 1
+      title: Password
+      always_show: true
+      airbyte_secret: true
+    username:
+      type: string
+      order: 0
+      title: Username
+    batchsize:
+      type: string
+      order: 3
+      title: batchsize
+      default: '100'
```

Comment:
Why are you exposing this variable to users? Is there a maximum allowed by the API?

Reply:
There is no limit set by the API but will want to control this to avoid overloading the API. 100 should be ok for most cases but might need to be tuned on busy systems to stay up to date vs to stress a source system.